### ISSUE TITLE
Compile error == operator overloading

### ISSUE THREAD

**Issue by thokag** [fuchsto/cpppc-18#8]
Beim compile des measurement Tests erscheint folgende Fehlermeldung:
/usr/include/gtest/gtest.h:1430:11: error: invalid operands to binary expression
      ('int' and 'const cpppc::Measurements<int>')
  if (lhs == rhs) {
      ~~~ ^  ~~~
Warum wird hier lhs als int interpretiert? Die Signatur meines ==-Operators sieht wie folgt aus:
edit:
`bool operator==(const self_t &rhs) const`
<del>//bool operator==(const vector &v1) const<del>

**Comment by jakubo87** [comment]
bist du dir sicher, dass es da um genau diese operation geht? ist nämlich etwas in gtest und die rechte Seite ist auch kein vektor...

**Comment by thokag** [comment]
stimmt, da hab ich schon einiges rumprobiert und nicht die eigentliche Signatur eingefügt.
 "bool operator==(const self_t &rhs) const"
 sollte da eigentlich stehen. Soweit ich das verstehe wird von gtest hier schon vorausgesetzt, dass man einen == operator breitstellt
